agent/engine/engine_unix_integ_test.go

```diff
@@ -1336,6 +1336,43 @@ func TestSwapConfigurationTask(t *testing.T) {
 	verifyTaskStoppedStateChange(t, taskEngine)
 }
 
+func TestLabels(t *testing.T) {
```

Comment:
i think we should maintain the test on windows as well? the functional test runs on both linux and windows

Reply:
Yes will move the test to `engine_integ_test.go`